src/js/integrations/debugsymbolicator.ts

```diff
@@ -86,8 +87,14 @@ export class DebugSymbolicator implements Integration {
     try {
       const symbolicateStackTrace = require("react-native/Libraries/Core/Devtools/symbolicateStackTrace");
       const prettyStack = await symbolicateStackTrace(stack);
+
       if (prettyStack) {
-        const stackWithoutInternalCallsites = prettyStack.filter(
+        let newStack = prettyStack;
+        if (prettyStack.stack) {
+          // This has been changed in an react-native version so stack is contained in here
+          newStack = prettyStack.stack;
+        }
```

Comment:
This is the important change, the stack variable moved in a react native version bump